README.md

```diff
@@ -1,2 +1,28 @@
-# vip-inheritance-matcher
-annotates VCF samples with denovo and possible compound flags and matching inheritance modes
+# vip inheritance matcher
+annotates VCF samples with denovo and possible compound flags and matching inheritance modes and genes.
+
+###Added Sample information
```

Comment:
Add input requirements with a link to tooling to make to make your VCF meet the requirements.